### ISSUE TITLE
Support for multiple sources

### ISSUE THREAD

**Issue by stefcameron** [pedronauck/docz#864]
Docz is awesome! I'm wanting to use it in a monorepo where our layout is such that we have `.mdx` files in (from the root):
```
- docs: high-level general doc pages in .mdx files
- pacakges/**: non-component packages that may contain .mdx files
- components/**: component packages that contain .mdx files
```
Unfortunately, the Dcoz config file, as of 1.2.0, doesn't appear to support this layout since the `src` property is a single string (based on published documentation).

My request is simply to support an optional array for the `src` property. If not a string, then it should be an array, whereby the array would contain multiple sources to be searched according to the `files` glob property.

**Comment by draperunner** [comment]
I guess you mean the `files` config, not `src`? What if you just don't have a `files` config specified? Docz should then search through the entire project for MDX files.

Anyways, `files` takes a glob pattern where the default is `'**/*.mdx'`. If you want to exclude other folders than `docs/`, `pacakges/` and `components/`, you could do this I guess: `files: '{docs,pacakges,components}/**/*.mdx'`.

**Comment by stefcameron** [comment]
@draperunner I did, in fact, mean the `src`, _not_ `files`, but didn't understand the full extent of the possibilities that the `files` glob was affording. A pattern like you suggested does work, so I will close this request. Thank you for your help! 😄

It's possible, though, that adding support for additional sources, and associated globs (which is slightly different from what I started out asking for), in the future might provide a more intuitive first-time experience, as well as more flexibility around the globbing since patterns can sometimes gets pretty gnarly.
